File: python/encryption/hashing.py
# hashing.py

import hashlib

# SHA-256 Hash function
def hash_sha256(text: str) -> str:
    return hashlib.sha256(text.encode()).hexdigest()

# MD5 Hash function (not recommended for sensitive data)
def hash_md5(text: str) -> str:
    return hashlib.md5(text.encode()).hexdigest()

if __name__ == "__main__":
    msg = "Hello, World!"
    print(f"SHA-256: {hash_sha256(msg)}")
    print(f"MD5: {hash_md5(msg)}")

File: js/encryption/hashing.js
// hashing.js

const crypto = require('crypto');

// Hashing function using SHA-256
function hash(text) {
  return crypto.createHash('sha256').update(text).digest('hex');
}

// Hashing function using MD5 (not recommended for sensitive data)
function hashMD5(text) {
  return crypto.createHash('md5').update(text).digest('hex');
}

module.exports = { hash, hashMD5 };
console.log("Hashing true")